src/qt/masternodelist.cpp

```diff
@@ -263,6 +270,13 @@ void MasternodeList::updateNodeList()
     ui->tableWidgetMasternodes->clearContents();
     ui->tableWidgetMasternodes->setRowCount(0);
     std::map<COutPoint, CMasternode> mapMasternodes = mnodeman.GetFullMasternodeMap();
+#if QT_VERSION < 0x050200
+    const QDateTime dateTime1 = QDateTime::currentDateTime();
+    const QDateTime dateTime2 = QDateTime(dateTime1.date(), dateTime1.time(), Qt::UTC);
+    int offsetFromUtc = dateTime1.secsTo(dateTime2);
+#else
+    int offsetFromUtc = QDateTime::currentDateTime().offsetFromUtc();
+#endif
```

Comment:
May be it is better to move this code to separate local static function instead of copy-pasting it in 2 places?

Reply:
Yes please, no code duplication if possible

Reply:
fixed